### PR TITLE
PP7941 / PP7940 - Lets users know that it is possible to find a payment by searching for its metadata

### PR DESCRIPTION
**Pull request by NathanD-GDS**
### Context
Pay is releasing the ability to search for payments by their custom metadata. This is a UI-only change - no API changes. We don't tend to exhaustively document UI processes in the tech docs but there is precedence for mentioning functionality.

### Changes proposed in this pull request
This change adds a sentence to the 'Add custom metadata' docs page to let users know this search is possible.

### CODE REVIEW

**Comment by NathanD-GDS** [comment]
Closed PR until publishing freeze is over.

File: source/custom_metadata/index.html.md.erb [hunk 1]
```diff
@@ -49,4 +49,4 @@ When you [get information about a single payment](/reporting/#get-information-ab
 
 The order of the parameters in the `metadata` object may be different to your original object.
 
-You can also sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see or download a payment's metadata.
+You can also sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see or download a payment's metadata. You can also search for specific payments by their metadata in the GOV.UK Pay admin tool.
```
Comment (PippaClarkGDS) [inline source/custom_metadata/index.html.md.erb:52]: You can [sign in to the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see or download a payment's metadata. You can also search for specific payments by their metadata in the GOV.UK Pay admin tool.

Comment (PippaClarkGDS) [reply source/custom_metadata/index.html.md.erb:52]: Looks good - one small tweak for the link. Also wondering if we can be more specific for 'search for specific payments by their metadata' ... what metadata? Can a user use any metadata to search for this?

**Comment by NathanD-GDS** [comment]
Additional changes have been made after Pippa's pre-i review.

**Review by stephencdaly — APPROVED** [review]

**Pull request merged**